doc/api_plugins.rst

```diff
@@ -62,6 +62,7 @@ When DNF CLI runs it loads the plugins found in the paths during the CLI's initi
   .. method:: transaction
 
     Plugin can override this. This hook is called immediately after a successful transaction.
+    Plugins that were removed or obsoleted by the transaction do not run the transaction hook.
```

Comment:
If I understand it correctly, I think it should read "... by the transaction _will_ not run ..."?